[2021-05-12 15:32:48,960] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: etl.removing_local_files 2021-05-12T17:00:00+00:00 [queued]>
[2021-05-12 15:32:48,966] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: etl.removing_local_files 2021-05-12T17:00:00+00:00 [queued]>
[2021-05-12 15:32:48,966] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 15:32:48,966] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2021-05-12 15:32:48,966] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 15:32:48,971] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): removing_local_files> on 2021-05-12T17:00:00+00:00
[2021-05-12 15:32:48,974] {standard_task_runner.py:52} INFO - Started process 2057 to run task
[2021-05-12 15:32:48,979] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'etl', 'removing_local_files', '2021-05-12T17:00:00+00:00', '--job-id', '183', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/var/folders/_f/vg3x3jmj23bcz6s88l7v6fb40000gn/T/tmppcdm6vg5', '--error-file', '/var/folders/_f/vg3x3jmj23bcz6s88l7v6fb40000gn/T/tmp_dcpjzcp']
[2021-05-12 15:32:48,981] {standard_task_runner.py:77} INFO - Job 183: Subtask removing_local_files
[2021-05-12 15:32:49,013] {logging_mixin.py:104} INFO - Running <TaskInstance: etl.removing_local_files 2021-05-12T17:00:00+00:00 [running]> on host 27.2.168.192.in-addr.arpa
[2021-05-12 15:32:49,036] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=udacity
AIRFLOW_CTX_DAG_ID=etl
AIRFLOW_CTX_TASK_ID=removing_local_files
AIRFLOW_CTX_EXECUTION_DATE=2021-05-12T17:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-12T17:00:00+00:00
[2021-05-12 15:32:49,037] {python.py:118} INFO - Done. Returned value was: None
[2021-05-12 15:32:49,040] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=etl, task_id=removing_local_files, execution_date=20210512T170000, start_date=20210512T183248, end_date=20210512T183249
[2021-05-12 15:32:49,076] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-12 15:32:49,117] {local_task_job.py:146} INFO - Task exited with return code 0
